### ISSUE TITLE
Documentatie: richtlijn over voortgang in meerstappenformulier herzien

### ISSUE THREAD

**Issue by jeffreylauwers** [nl-design-system/documentatie#3600]
## TL;DR

In onze documentatie staat de [richtlijn: Geef aan hoeveel stappen er zijn en in welke stap de gebruiker zich bevindt](https://nldesignsystem.nl/richtlijnen/formulieren/meerdere-stappen/voortgang-tonen/). Op dit moment staat daar:

> Als een formulier meerdere stappen heeft, is het noodzakelijk dat een gebruiker weet in welke stap ze zich bevindt en hoeveel stappen er nog volgen. Dit geeft duidelijkheid en voorkomt onzekerheid bij de gebruiker.

Deze richtlijn lijkt te rigide. Op basis van de richtlijn bij GOV.UK en inzichten uit de onderzoeken van Den Haag en Utrecht kunnen we stellen dat het niet altijd noodzakelijk is. Maar dat dit een hulpstuk kan zijn wanneer uit onderzoek blijkt dat daar behoefte aan is. Daarbij zouden we moeten meenemen dat het aantal stappen ook conditioneel kan zijn.

## Acties

- Afstemmen of wij ons als kernteam in deze aanpassing kunnen vinden. Zie 'Notities' voor aanvullende informatie.
- Richtlijn aanpassen.
- Joshua informeren over de aanpassing.

## Acceptatiecriteria

- [x] Het aanpassen van de richtlijn is besproken met de nodige specialismes uit het kernteam.
- [x] De richtlijn is aangepast.

## Notities

- [Richtlijn: Geef aan hoeveel stappen er zijn en in welke stap de gebruiker zich bevindt](https://nldesignsystem.nl/richtlijnen/formulieren/meerdere-stappen/voortgang-tonen/)
- [Richtlijn op GitHub](https://github.com/nl-design-system/documentatie/blob/main/docs/richtlijnen/formulieren/multistep/1-step-count/_guideline.md)
- [Bericht van Joshua op Slack](https://codefornl.slack.com/archives/C01D78C2E4E/p1768213116400529) waar in het draadje ook goeie punten worden genoemd.
- [GitHub Discussion: Voortgang indicatie - Meerstappenformulier](https://github.com/orgs/nl-design-system/discussions/369)
- [Question pages - Using progress indicators - GOV.UK](https://design-system.service.gov.uk/patterns/question-pages/#:~:text=Using%20progress%20indicators)
